obyde/__init__.py

```diff
@@ -106,9 +126,10 @@ def rewrite_links(content, dated_file_index, asset_index, relative_asset_path_pr
                 break
         if not written:
             link_name_slug = slugify_md_filename(link_target)
-            dated_name, _, _ = dated_file_index[link_name_slug]
-            rewritten = rewritten.replace(
-                link, f'[{link_text}]({generate_post_link(dated_name, post_link_mode)})')
+            dated_name, _, _ = dated_file_index.get(link_name_slug, (None, None, None))
```

Comment:
Right now, if a file has a `[[` link to a file that doesn't exist yet, obdye errors out. This change makes it skip replacing the link instead.